libraries/ms-helper/inc/thermistor.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+// Thermistor Reading interface for NXRT15XH103FA1B___
```

Comment:
The part number does not include the underscores. There's no need to link the actual datasheet. You should mention that it's a 10k NTC thermistor.

Reply:
The data sheet is one of the sub headings on the product page's link. I can add the direct link to the  datasheet if that works better?

Reply:
No, there's no need to link the datasheet. People can find it themselves using the part number.